packages/button/test/button.test.ts

```diff
@@ -97,6 +97,41 @@ describe('Button', () => {
         expect(el).to.not.be.undefined;
         await expect(el).to.be.accessible();
     });
+    it('has a stable/predictable `updateComplete`', async () => {
+        const test = await fixture<HTMLDivElement>(
+            html`
+                <div></div>
+            `
+        );
+
+        let keydownTime = -1;
+        let updateComplete1 = -1;
+        let updateComplete2 = -1;
+
+        const el = document.createElement('sp-button');
+        el.autofocus = true;
+        el.addEventListener('keydown', () => {
+            keydownTime = performance.now();
+        });
+        el.updateComplete.then(() => {
+            updateComplete1 = performance.now();
+        });
+        el.updateComplete.then(() => {
```

Comment:
These two `updateComplete` promises can potentially be triggered at the same time

Reply:
That's actually the goal. The previous code positioned the first to happen _after_ the second 😞